docs/UserGuide.md

```diff
@@ -168,7 +166,27 @@ Examples:
 * `edit 2 n/Betsy Crower t/` Edits the name of the 2nd student to be `Betsy Crower` and clears all existing tags.
 * `edit 3 sch/NJC stream/` Edits the school of the 3rd student to be `NJC` and clears academic stream data.
 
-### Finding students by fields: `find`
+#### Deleting a person: `delete`
+
+Deletes the specified person from the address book.
+
+Format: `delete INDEX`
+
+* Deletes the person at the specified `INDEX`.
+* The index refers to the index number shown in the displayed person list.
+* The index **must be a positive integer** 1, 2, 3, …​
+
+Examples:
+* `list` followed by `delete 2` deletes the 2nd person in the address book.
+* `find Betsy` followed by `delete 1` deletes the 1st person in the results of the `find` command.
+
+#### Listing all students: `list`
+
+Shows a list of all students in the address book.
+
+Format: `list`
```

Comment:
personally i feel like `list` and `view` should be together since they are both viewing functions

Reply:
can can I'll move `view` to before `list`